tests/unit/records/marshmallow/jobs/test_base.py

```diff
@@ -6,19 +6,19 @@
 # the terms of the MIT License; see LICENSE file for more details.
 
 from inspirehep.records.api import JobsRecord
-from inspirehep.records.marshmallow.jobs import JobsMetadataRawFieldsSchemaV1
+from inspirehep.records.marshmallow.jobs.base import JobsRawSchema
 
 
 def test_jobs_serializer_should_serialize_whole_basic_record():
-    schema = JobsMetadataRawFieldsSchemaV1()
+    schema = JobsRawSchema()
     data = {
-        '$schema': 'http://localhost:5000/schemas/records/jobs.json',
-        '_collections': ['Jobs'],
-        'deadline_date': '1996-11-15',
-        'description': 'Join us!',
-        'position': 'staff',
-        'regions': ['Europe'],
-        'status': 'closed'
+        "$schema": "http://localhost:5000/schemas/records/jobs.json",
+        "_collections": ["Jobs"],
+        "deadline_date": "1996-11-15",
+        "description": "Join us!",
+        "position": "staff",
+        "regions": ["Europe"],
+        "status": "closed",
```

Comment:
are these the all the schema fields?

Reply:
not the whole record. Actually this is a copy paste and  I think it's useless since `RecordBaseSchema` is tested via unit tests (so that we are sure it includes everything) and jobs serialiser tested via integration tests.